### PR TITLE
Add client management module code

### PR DESCRIPTION
**Pull request by gouravmalviya061290**
Add client management module code

### CODE REVIEW

**Review by ramverma-github — COMMENTED** [review]
Please resolve the comments code.

File: yota-api/pom.xml [hunk 1]
```diff
@@ -53,7 +53,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-
+<!-- https://mvnrepository.com/artifact/org.apache.poi/poi -->
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/pom.xml]: Done

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.web.bind.annotation.RestController;
+import com.yash.yotaapi.domain.CreateClientQuestion;
+import com.yash.yotaapi.service.ClientQuestionBankService;
+import com.yash.yotaapi.util.FieldErrorValidationUtillity;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@CrossOrigin("*")
+@RestController
+@Tag(name = "Client Question Bank Controller", description = "Controller for Client Question Bank")
+@RequestMapping("/yota/api/clientQuestion")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+public class ClientQuestionBankController {
+
+	@Autowired
+	ClientQuestionBankService clientQuestionBankService;
+	@Autowired
+	private FieldErrorValidationUtillity mapValidationErrorService;
+
+	@PostMapping("/add")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+			BindingResult result) {
+		ResponseEntity<?> errmap = mapValidationErrorService.validationError(result);
+		if (errmap != null) {
+			return errmap;
+		}
+		CreateClientQuestion savedQuestion = clientQuestionBankService.saveOrUpdate(question);
+		return new ResponseEntity<CreateClientQuestion>(savedQuestion, HttpStatus.CREATED);
+	}
+
+	@GetMapping("/getAllQuestion")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/ClientQuestionBankController.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/controller/CreateClientController.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.yash.yotaapi.domain.CreateClient;
+import com.yash.yotaapi.service.CreateClientService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@CrossOrigin("*")
+@RestController
+@Tag(name = "Client Question Bank Controller", description = "Controller for Client Question Bank")
+@RequestMapping("/yota/api/client")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/CreateClientController.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/controller/CreateClientController.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+public class CreateClientController {
+
+	@Autowired
+	private CreateClientService clientQuestionBankService;
+
+	@PostMapping("/createclient")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/controller/CreateClientController.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/domain/CreateClient.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Data
+@Table(name = "Create_Client")
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/domain/CreateClient.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/domain/CreateClient.java [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+public class CreateClient {
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/domain/CreateClient.java]: done

File: yota-api/src/main/java/com/yash/yotaapi/service/CreateClientService.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
+package com.yash.yotaapi.service;
+
+import com.yash.yotaapi.domain.CreateClient;
+import com.yash.yotaapi.domain.CreateClientQuestion;
+import com.yash.yotaapi.domain.Question;
+
+public interface CreateClientService {
```
[the remark this answers is not in the document]
Comment (gouravmalviya061290) [reply yota-api/src/main/java/com/yash/yotaapi/service/CreateClientService.java]: done

**Review by ramverma-github — APPROVED** [review]
The changes look good.

**Pull request merged**